frontend/lib/src/components/elements/DeckGlJsonChart/withMapboxToken/withMapboxToken.tsx

```diff
@@ -96,6 +141,19 @@ const withMapboxToken =
         }
       }
 
+      public componentDidMount(): void {
+        const { mapboxToken } = this.context.hostConfig
```

Comment:
I think we need to give priority to the user-configured token, so we need to do something like this here:

```suggestion
        const mapboxToken = this.props.sessionInfo.current.userMapboxToken || this.context.hostConfig.mapboxToken
```

And only do the token fetching in the `initMapboxToken` method (this function could also be renamed in that case).

Reply:
Oops yep. I'm so used to HostConfig being priority 😓